src/Metrics.tcl

```diff
@@ -123,9 +131,16 @@ proc report_design_area_metrics {args} {
     set inst_area  [expr $wid * $ht]
     set total_area [expr $total_area + $inst_area] 
     set num_insts [expr $num_insts + 1]
-    if { [string match [$inst_master getType] "BLOCK"] } {
+
```

Comment:
The area calculation is problematic for any non-rectangular masters. It won't come up often but it might be good to leave a TODO because I don't think there's a `getArea()` function for `dbMaster`.